client-cli/src/command/transaction_command.rs

```diff
@@ -131,6 +133,55 @@ pub enum TransactionCommand {
         )]
         tx: String,
     },
+    #[structopt(
+        name = "build",
+        about = "build a raw transfer transaction for offline wallet"
+    )]
+    Build {
+        #[structopt(
+            name = "wallet name",
+            short = "n",
+            long = "name",
+            help = "Name of wallet"
+        )]
+        name: String,
+    },
+    #[structopt(
+        name = "sign",
+        about = "sign a raw transfer transaction on offline wallet"
+    )]
+    Sign {
+        #[structopt(
+            name = "wallet name",
+            short = "n",
+            long = "name",
+            help = "Name of wallet"
+        )]
+        name: String,
+        #[structopt(
+            name = "tx_unsigned",
+            short = "t",
+            long = "tx_unsigned",
+            help = "base64 encoded raw transaction info"
```

Comment:
how about adding option for just json?

Reply:
base58 is better I think.

Reply:
base58 was meant for potential human input -- this is more for copy-paste.
base64 is ok for that.

another option is just to dump to a binary file and provide a path to that file